.github/workflows/publish.yml

```diff
@@ -40,8 +40,51 @@ jobs:
       - name: Install mdbook
         uses: ./.github/workflows/install-mdbook
 
+      - name: Install mdbook-pandoc
+        run: curl --proto '=https' --tlsv1.2 -LsSf https://github.com/max-heller/mdbook-pandoc/releases/download/v0.2.0/mdbook-pandoc-installer.sh | sh
```

Comment:
This could be `cargo install mdbook-pandoc` instead: we cache the Rust binaries installed, so it'll be pretty quick after the first run.

(I try to avoid having long as "mysterious" command lines in the actions — I don't know what `-LsSf` means and I prefer not having to know about it).